### ISSUE TITLE
Automatically Disable leaderboard snakes that timeout on start multiple times.

### ISSUE THREAD

**Issue by cbinners** [battlesnakeio/roadmap#265]
Maybe send an email to the owner indicating the new status?

**Comment by dlsteuer** [comment]
Yes we should do something about leaderboard snakes that don't respond. This is larger then just the leaderboard, and could be any snake in a game.

**Comment by DirkGehlich** [comment]
The leaderboard is not reachable at all for me since some days. So I agree that something should be done.

**Comment by DirkGehlich** [comment]
I think this issue can be closed, as it has been fixed by battlesnakeio/play#109
